docs/proposal-007-instance-selector.md

```diff
@@ -0,0 +1,269 @@
+# EC2 Instance Selector Support for Nodegroups
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+- [Proposal](#proposal)
+- [Design](#design)
+  - [eksctl create cluster](#eksctl-create-cluster)
+  - [eksctl create nodegroup](#eksctl-create-nodegroup)
+  - [API Changes](#api-changes)
+  - [Caveats](#caveats)
+<!-- /toc -->
+
+## Summary
+
+This proposal adds support for the EC2 instance selector for both managed and unmanaged nodegroups.
+
+
+## Motivation
+
+With support for multiple instance types and Spot instances for EKS Managed Nodegroups, users can create Spot instances where EKS will configure and launch an Autoscaling Group of Spot instances following  Spot best practices. Users can set `managedNodeGroup.spot` and supply a list of instance types in `managedNodeGroup.instanceTypes` to instruct eksctl to create a managed nodegroup using Spot instances.
+
+It is a Spot best practice to use multiple instance types, but with over 270 EC2 instance types, users have to spend time figuring out which instance types would be well suited for their nodegroup.
```

Comment:
Same call out here - this is an issue in general, not just for those using Spot.